packages/style-utils/src/createStyledComponent.js

```diff
@@ -15,7 +15,12 @@
  */
 
 /* @flow */
-import glamorous from 'glamorous';
+import glamorous from "glamorous";
+
+// $FlowFixMe
+glamorous.config.useDisplayNameInClassName = NODE_ENV === "development"
```

Comment:
I think this setting is something that should be left up to the consumer to configure in their own app.  I also believe that it will be the default in the future.  I experimented using it previously and really didn't feel like it added any value.